installers/linux/deb/kvm2_deb_template/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Package: docker-machine-driver-kvm2
+Version: --VERSION--
+Section: base
+Priority: optional
+Architecture: amd64
+Depends: libvirt0 (>= 1.3.1)
+Recommends: minikube
+Maintainer: Aaron Prindle <aaprindle@gmail.com>
+Description: Machine driver for KVM
+ Minikube uses Docker Machine to manage the Kubernetes VM so it benefits
```

Comment:
I prefer 'minikube' to 'Minikube'. We are mini, after all. :)

Reply:
Weird to start a sentence with a lower case letter, but OK